cohere/client.py

```diff
@@ -232,11 +232,28 @@ def unstable_extract(self, examples: List[ExtractExample], texts: List[str]) ->
 
         return Extractions(extractions)
 
-    def summarize(self, text: str, length: str = "normal", useBaseline: bool = False) -> Summary:
+    def summarize(
+        self, 
+        text: str, 
+        length: str = "normal", 
+        useBaseline: bool = False,
+        max_tokens: int = 120,
+        temperature: float = 1.0,
+        k: int = 1,
+        p: float = 1,
+        frequency_penalty: float = 0.0,
+        presence_penalty: float = 0.0
```

Comment:
@choicallum don't duplicate defaults here. If we change defaults in the service they'd be ignored by SDK users